MacLinux/README.txt

```diff
@@ -45,7 +46,7 @@ Rules
 - Tha lambda function $RULE_NAME.zip needs to be in the same directory as the createRule script
 - Create $RULE_NAME.zip from files in $RULE_NAME directory
 
-Test
+Test // DO NOT USE, STILL BETA

```

Comment:
Hi Greg, the profile can be made optional in the next version. Agreed. I just wanted to have it. We can just put "default" as parameter for now. Overall for the compliance-as-code, people usually have their config, so I prefer to have vs not having it.